### PR TITLE
[IMP] remove tmp_val[1-5]

### PR DESCRIPTION
**Pull request by gurneyalex**
* migrate Login / Logout scenario to use tmp_values
* rewrite the helper properties to store these values in the
  serialized fields tmp_values
* add a computed field to display the tmp_values in a table in
  the hardware form view.

### CODE REVIEW

**Review by benwillig — APPROVED** [review]

File: stock_scanner/models/scanner_hardware.py [hunk 1]
```diff
@@ -173,61 +153,85 @@ def _colors_get(self):
     tmp_values = fields.Serialized(
         readonly=True
     )
+    tmp_values_display = fields.Html(
+        compute='_compute_tmp_values_txt',
```
Comment (simahawk) [inline stock_scanner/models/scanner_hardware.py]: nitpicking: I'd expect `_compute_tmp_values_display`

File: stock_scanner/views/scanner_hardware.xml [hunk 2]
```diff
@@ -55,13 +55,9 @@
...
                     <group colspan="4">
-                        <separator string="Temporary values" colspan="2"/>
-                        <field name="tmp_val1"/>
-                        <field name="tmp_val2"/>
-                        <field name="tmp_val3"/>
-                        <field name="tmp_val4"/>
-                        <field name="tmp_val5"/>
-                    </group>
+                      <separator string="Temporary values" colspan="2"/>
+                      <field name="tmp_values_display"/>
```
Comment (simahawk) [inline stock_scanner/views/scanner_hardware.xml]: `nolabel=1` ?

**Review by simahawk — APPROVED** [review]
non blocking comments

File: stock_scanner/models/scanner_hardware.py (hunk 1 above)
Comment (gurneyalex) [reply stock_scanner/models/scanner_hardware.py]: yeah... I renamed the field but forgot to propagate in the compute method name...

**Review by lmignon — APPROVED** [review]
Great!

**Review by guewen — APPROVED** [review]
Great

**Comment by gurneyalex** [comment]
build is green : https://travis-ci.org/gurneyalex/stock-logistics-barcode/builds/359234745

**Comment by Garamotte** [comment]
Thanks a lot :)

**Pull request merged**